hash/ibm5170.xml

```diff
@@ -13985,6 +13994,66 @@ Eye of the Beholder III: Assault on Myth Drannor
 		</part>
 	</software>
 
+	<software name="fpsbas94">
+		<description>Front Page Sports: Baseball '94</description>
+		<year>1994</year>
+		<publisher>Sierra On-Line</publisher>
+		<notes><![CDATA[
+Bad Dump: the disks has an accessed file record
+]]></notes>
+		<info name="copy_protection" value="Manual required" />
+		<info name="developer" value="Dynamix" />
+		<info name="version" value="1.00W" />
+		<part name="flop1" interface="floppy_3_5">
+			<feature name="part_id" value="Disk 1" />
+			<dataarea name="flop" size="1474560">
+				<rom name="Front Page Sports - Baseball 94 [Sierra] [1994] [3.5HD] [disk 1 of 4].img" size="1474560" crc="21df8fa1" sha1="4f697050eb3169c77b27022464c05445111641f1" status="baddump"/>
+			</dataarea>
+		</part>
+		<part name="flop2" interface="floppy_3_5">
+			<feature name="part_id" value="Disk 2" />
+			<dataarea name="flop" size="1474560">
+				<rom name="Front Page Sports - Baseball 94 [Sierra] [1994] [3.5HD] [disk 2 of 4].img" size="1474560" crc="9192a82b" sha1="561ae9281b4d5d2256000a2ae72b17734e6ccc5b" status="baddump"/>
+			</dataarea>
+		</part>
+		<part name="flop3" interface="floppy_3_5">
+			<feature name="part_id" value="Disk 3" />
+			<dataarea name="flop" size="1474560">
+				<rom name="Front Page Sports - Baseball 94 [Sierra] [1994] [3.5HD] [disk 3 of 4].img" size="1474560" crc="821c2c9a" sha1="c392aed95ae839d261e07aeffe2af8d1e7ef8aad" status="baddump"/>
+			</dataarea>
+		</part>
+		<part name="flop4" interface="floppy_3_5">
+			<feature name="part_id" value="Disk 4" />
+			<dataarea name="flop" size="1474560">
+				<rom name="Front Page Sports - Baseball 94 [Sierra] [1994] [3.5HD] [disk 4 of 4].img" size="1474560" crc="83c0ee95" sha1="000fe3f30743e026319ace6c2b2044c0fe02bd90" status="baddump"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="fpsbas94exp">
+		<description>Front Page Sports: Baseball '94 - Legends and Stadiums expansion</description>
```

Comment:
Data disks goes as `cloneof` the intended SW they refers to.

Reply:
> Data disks goes as `cloneof` the intended SW they refers to.

I'm not sure about that. A few time ago, the add-ons were not considerated as `cloneof`, only a different set from the parent. As it already happens with the "Star Wars: X-Wing - B-Wing Expansion" and "Star Wars: X-Wing - Imperial Pursuit" they aren't related directly with the game "Star Wars: X-Wing - Space Combat Simulator". The same happens with "Aces of the Pacific Expansion Disk - WWII: 1946" and "Aces of the Pacific" or with "Alone in the Dark 2 - Secret Doors" and "Alone in the Dark 2".
There are more cases like that.
I can change it, but we must be sure that is the way it should be.

Reply:
I checked prior to commenting thru the hash folder: it's mostly done with a parent-clone relationship, and it honestly make sense this way rather than being separated parents.